pkg/minikube/registry/drvs/docker/docker.go

```diff
@@ -111,7 +111,11 @@ func status() registry.State {
 			return registry.State{Error: newErr, Installed: true, Healthy: false, Fix: "Add your user to the 'docker' group: 'sudo usermod -aG docker $USER && newgrp docker'", Doc: "https://docs.docker.com/engine/install/linux-postinstall/"}
```

Comment:
I suggest moving all of the string matching `if` statements into a separate function to improve readability